website/content/docs/commands/init.mdx

```diff
@@ -66,6 +66,16 @@ string](/docs/templates/hcl_templates/blocks/packer#version-constraints). The
 plugins will be installed in the [Plugin
 Directory](/docs/configure#packer-s-plugin-directory).
 
+### Implicit plugin requirements
+
+When running `packer init`, if Packer logs a warning that tells you to add a
+`packer.required_plugin` block to your config file, this is because this plugin
+was moved out of the Packer binary and you should consider using that block.
+This is part of a set of breaking changes made to decouple Packer releases from
+a Plugin release. These suggestion are manually added each time we move a plugin
+out of the Packer core. These should happen only starting from version `2.0` of
+Packer.
```

Comment:
I thought we were going to add a redirect every time we have a valid release for a plugin we've broken out. With that said, I was expecting the implicitly required plugins to start working to all plugins after 1.8.0.

Reply:
Because of this code here:

https://github.com/hashicorp/packer/blob/7ccfe38475fcd153571e16382eebf32125ad93b6/hcl2template/types.required_plugins.go#L102-L113

This is not possible as the plugin is already present in core. There's no simple way to say 'ignore-core-plugins', unless we add an option `packer init --ignore-core-plugins .` ?

Reply:
As I understood these are the plugins that are still in the core, including the vendored ones, as docker is currently. But for 1.8.0, are we keeping broken out plugins vendored? I thought we weren't.

Reply:
Allowing to now pre-load plugins ( with or without the `--ignore-core-plugins` option ) is doable but it would require changing a little bit of the initialisation of Packer plugins that is currently done in `main.go`, to hand it over to command package. I think this would be sort of more idiomatic but take some more time. What do you think @hashicorp/packer ? Should I do that ?

Reply:
Yes we are keeping them vendored for now. 2.0 is like a major breaking change as we will un-vendor those

Reply:
I'm a bit confused because the `--ignore-core-plugins` is not really related to what I'm saying/thinking. I'm considering when the plugin is not in the core. I thought we were going to stop vendoring plugins on 1.8.0.

Reply:
> I'm a bit confused because the --ignore-core-plugins is not really related to what I'm saying/thinking. I'm considering when the plugin is not in the core. I thought we were going to stop vendoring plugins on 1.8.0.

Ah, sorry for the miscommunication, I thought you were suggesting to make it work for 1.8, and was already one step ahead thinking on how to implement that.

Reply:
( Moved to slack ) Moss is right, we meant to bring this with 1.8 ! 
```suggestion
out of the Packer core. These should happen only starting from version `1.8` of
Packer.
```